automation/roles/common/defaults/main.yml

```
@@ -720,3 +720,25 @@ netdata_conf:
   # You can fine-tune retention for each tier by setting a time limit or size limit. Setting a limit to 0 disables it.
 # More options you can specify in the roles/netdata/templates/netdata.conf.j2
 # https://learn.netdata.cloud/docs/netdata-agent/configuration
+
+### Metric ###
+with_metric_exporters: false # or true
+
+# Node Exporter
+node_exporter_bin_path: /opt/prometheus/node-exporter
+node_exporter_host: ""
+node_exporter_port: 9100
+node_exporter_options: ""
+
+# Postgres Exporter
+postgres_exporter_bin_path: /opt/prometheus/postgres-exporter
+postgres_exporter_host: ""
+postgres_exporter_port: 9187
+postgres_exporter_db: postgres
+postgres_exporter_user: postgres
+postgres_exporter_data_source_name: "user={{ postgres_exporter_user }} dbname={{ postgres_exporter_db }} host={{ postgresql_unix_socket_dir }} sslmode=disable"
+postgres_exporter_query_filenames:
+  - queries-default.yml
+  # - queries-pg_stat_statements.yml
```

```suggestion
  - queries-pg_stat_statements.yml
```

I suggest setting pg_stat_statements by default

```yaml
postgresql_extensions:
  - { ext: "pg_stat_statements", db: "postgres" }
```